pkgs/top-level/python-packages.nix

```diff
@@ -3723,6 +3723,8 @@ in {
 
   HTSeq = callPackage ../development/python-modules/HTSeq { };
 
+  httpagentparser = callPackage ../development/python-modules/httpagentparser { };
+
```

Comment:
Please add the entry to a "alphabetically better position" please.

Reply:
ups. wrong line..